packages/body/tests/site/index.html

```diff
@@ -117,7 +117,7 @@
 
 	<p>Back to a paragraph.</p>
 
-	<hr />
+	<hr>
```

Comment:
Fair ’nuff — still valid being self-closing (being a void tag) but might as well remain consistent with the general approach html is going in.